install/installer/traf_sqgen

```diff
@@ -0,0 +1,96 @@
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+export TRAF_CONFIG=/etc/trafodion/trafodion_config
+
+source $TRAF_CONFIG
+
+STARTING_NODE=$(hostname)
+
+#==========================================
+echo "***INFO: starting sqgen" | tee -a $INSTALL_LOG
+cd $SQ_ROOT/sql/scripts
+$SQ_ROOT/sql/scripts/sqgen
+SQGEN_RC=$?
+if [ "$SQGEN_RC" != "0" ]; then
+    echo "***ERROR: sqgen failed with RC=$SQGEN_RC. Check install log file for details." | tee -a $INSTALL_LOG
+    exit -1;
+fi
+#==========================================
+
+if [ "$node_count" -ne "1" ]; then
+    echo "***INFO: copying $HOME/sqcert directory to all nodes" | tee -a $INSTALL_LOG
+    $PDCP $ALL_NODES -x $HOSTNAME -r $HOME/sqcert $HOME
+fi
+
+#==========================================
+
+if [ "$all_node_count" -ne "1" ]; then
+   echo "***INFO: copying install to all nodes" | tee -a $INSTALL_LOG
+   $PDCP $ALL_NODES -x $HOSTNAME -r $SQ_ROOT $SQ_ROOT/..
+
+   if [ "$?" != "0" ]; then
+      echo "***ERROR: Unable to copy Trafodion install to all machines in this cluster.  Check install log files for details." | tee -a $INSTALL_LOG
+      exit -1;
+   fi
+fi
+
+#=========================================
+#Copying traf_authentication_conf to scripts directory
+if [[ "$LDAP_SECURITY" == "Y" ]]; then
+   if [ "$all_node_count" -ne "1" ]; then
+      echo "***INFO: Copying $LDAP_AUTH_FILE to all nodes" | tee -a $INSTALL_LOG
+      $PDCP $ALL_NODES -r $HOME/$LDAP_AUTH_FILE $SQ_ROOT/sql/scripts/traf_authentication_config
+      if [ "$?" != "0" ]; then
+        echo "***ERROR: Unable to copy $LDAP_AUTH_FILE to all machines in this cluster.  Check install log files for details." | tee -a $INSTALL_LOG
+         exit -1;
+      fi
+   else
+      echo "***INFO: Copying $LDAP_AUTH_FILE to scripts directory." | tee -a $INSTALL_LOG
+      cp -rf $HOME_DIR/$TRAF_USER/$LDAP_AUTH_FILE $SQ_ROOT/sql/scripts/traf_authentication_config
+   fi
+
+   #Check traf_authentication_config for errors
+   echo "***INFO: Checking LDAP Configuration file for errors."
+   ldapconfigcheck -file $SQ_ROOT/sql/scripts/traf_authentication_config
+   if [ "$?" != "0" ]; then
+      echo "***ERROR: traf_authentication_config not configured correctly."
+      echo "***ERROR: Install will continue WITHOUT simple security turned on."
+      echo "***ERROR: Please review wiki for manual steps to setup simple security."
```

Comment:
Because security can be enabled after the installation, the install should continue.